aspnetcore/security/authentication/azure-ad-b2c/sample/Startup.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+#region snippet_NewUsings
+using Microsoft.Identity.Web;
+using Microsoft.Identity.Web.UI;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+#endregion
+
+namespace azure_ad_b2c
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        #region snippet_ConfigureServices
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
+                .AddMicrosoftIdentityWebApp(Configuration.GetSection("AzureAd"));
+
+            services.AddAuthorization(options =>
+            {
+                // By default, all incoming requests will be authorized according to the default policy
```

Comment:
```suggestion
                // By default, all incoming requests will be authorized according to 
                // the default policy
```

Gets rid of horizontal scroll bar on tablets 
![image](https://user-images.githubusercontent.com/3605364/126788592-c8817c3e-0ccf-448c-9c1a-0e05b7cfa9a0.png)